hummingbot/client/ui/__init__.py

```diff
@@ -1,10 +1,19 @@
+import os
+import sys
+from os.path import dirname, join, realpath
+from typing import Optional, Type
+
 from prompt_toolkit.shortcuts import input_dialog, message_dialog
 from prompt_toolkit.styles import Style
-from os.path import join, realpath, dirname
 
+from hummingbot.client.config.conf_migration import migrate_configs
+from hummingbot.client.config.config_crypt import BaseSecretsManager, store_password_verification
 from hummingbot.client.config.global_config_map import color_config_map
+from hummingbot.client.config.security import Security
+from hummingbot.client.settings import CONF_DIR_PATH
+
+sys.path.insert(0, realpath(join(__file__, "../../../")))
```

Comment:
Hmmm. i think this is done to resolve paths marked simply as `"/config"`. The new approach we're introducing to path specification (i.e. using `root_path`), makes this unnecessary, but I will keep it in case I've overlooked some legacy path definitions and I generally can't see it hurting to have as precaution.

I will change it to use `root_path` instead.